src/external/os/Earthfile

```diff
@@ -62,3 +62,9 @@ ADDUSER:
     USER ${user}
     ENV HOME=/home/${user}
     WORKDIR ${workdir}
+
+FIXUSER:
+    FUNCTION
+    ARG user
+    ARG workdir=/home/${user}
+    RUN rosdep fix-permissions
```

Comment:
@marcoshuck have you tried this out? Where is a `workdir` used? I would've expected `rosdep` to require `sudo`

Reply:
Also, missing EOF newline.

Reply:
I ended up rewriting the code in this function and forgot to remove the workspace, I ran the whole build and it seemed to have worked as intended, can try running it again from scratch

Reply:
+1 to adding CI, never got around to do it.

Reply:
I re-ran the build without `sudo` and worked as intended.

Reply:
Not sure if it's worth removing `rosdep fix-permissions`, since we changed the base, the issue was probably fixed somewhere else.